app/src/main/java/dev/andrylat/raqimbek/bankingutils/BankingUtilsApp.java

```diff
@@ -0,0 +1,26 @@
+package dev.andrylat.raqimbek.bankingutils;
+
+public class BankingUtilsApp {
+    private static CardValidationUserInteraction userInteraction = new CardValidationUserInteraction();
+    private static CardValidator cardValidator = new CardValidator();
+    private static PaymentSystemDeterminer paymentSystemDeterminer = new PaymentSystemDeterminer();
+
+    public static void main(String[] args) {
+        run();
+    }
+
+    public static void run() {
+    	userInteraction.write("Hello. Enter card number for validation:", System.out);
+    	userInteraction.read();        
```

Comment:
Bad Idea. You should call the method and receive result. Why do you need this state? You can't call "read()" method twice in a row and it is not easy to use. 

As I said: you call method and in the best you should provide parameters as inputs and receive valu as output. It is the best way to implement methods.